synapse/storage/registration.py

```diff
@@ -337,6 +350,16 @@ def is_support_user(self, user_id):
         )
         return res
 
+    def is_real_user_txn(self, txn, user_id):
+        res = self._simple_select_one_onecol_txn(
+            txn=txn,
+            table="users",
+            keyvalues={"name": user_id},
+            retcol="user_type",
+            allow_none=True,
+        )
+        return True if res is None or res == "" else False
```

Comment:
```suggestion
        return res is None or res == ""
```

Reply:
Also, why do we have empty strings here?

Reply:
> Also, why do we have empty strings here?

Can we guarantee `user_type` is always going to be `null` if not set to something? I would assume there is the possibility it will be an empty string. But can modify if not.

Reply:
Somebody would have had to explicitly set it to the empty string, at which point they could have equally set it to something weird anyway. I'd just handle it as `res is None` tbh

Reply:
@erikjohnston fixed